ui/admin/tests/e2e/README.md

```diff
@@ -35,10 +35,11 @@ You will need [Homebrew](https://brew.sh/) install. For secure persisting your S
 
 ### Accesses
 
-If you are missing any acccess, requested [through the IT service catalog](https://hashicorp.freshservice.com/support/catalog/items).
-
 - Doormat account: `boundary_team_acctest_dev`.
+  - Request access through Doormat.
 - HCP account: you will need to use Terraform cloud to spin up enos.
+- app.terraform.io org: `hashicorp-qti`.
```

Comment:
shouldn't we create our own TF token?  (maybe mention that and make this optional?)

Reply:
Update: The quality team confirmed that we need to be part of this org. Otherwise, you can't use the TF token that you create to access Enos resources.

Reply:
Do we have a shared token in our pw manager or should we always be requesting access to this org?

Reply:
Good question! It is [mentioned](https://github.com/hashicorp/boundary/blob/main/enos/enos.vars.hcl#L23) that a token is stored in 1password (tho not sure in which boundary vault). However, was told that it's recommended for folks to make individual tokens for security purposes.